### ISSUE TITLE
Malformed XML attribute names in testsuites element of JUnit reports

### ISSUE THREAD

**Issue by LogMANOriginal** [JKISoftware/Caraya#177]
JUnit reports generated with version 1.4.0 of Caraya contain malformed XML attribute names in the `testsuites` element.

```xml
<testsuites "framework-name"="Caraya" "framework-version"="1.4.0.142">
```

Double-quotes are not allowed in attribute names. The correct format is:

```xml
<testsuites framework-name="Caraya" framework-version="1.4.0.142">
```

**Comment by francois-normandin** [comment]
Thanks.
Will look into this urgently.

**Comment by francois-normandin** [comment]
I added another unit test to replace the disallowed space in the attribute name. Technically, we should validate a whole bunch of disallowed characters, but spaces are the most likely instance of a disallowed char.

Before Fix: 
![image](https://github.com/JKISoftware/Caraya/assets/11728548/bb5fbc9a-8007-4a2c-a231-641cdecf05a2)

Source below, to showcase the new Properties feature:
![image](https://github.com/JKISoftware/Caraya/assets/11728548/90ef121f-344e-4b8a-a0a3-e58bf6fbd340)

**Comment by francois-normandin** [comment]
![image](https://github.com/JKISoftware/Caraya/assets/11728548/4fa43982-8610-41b1-84c6-4e7596da4082)

**Comment by francois-normandin** [comment]
Released 1.4.1 with updated Release Notes. Thanks @LogMANOriginal (again) for being an awesome Caraya tester!

https://github.com/JKISoftware/Caraya/releases/tag/1.4.1
